app/models/katello/content_view_component.rb

```diff
@@ -31,33 +31,33 @@ def ensure_valid_content_view
       return unless view
 
       if content_view_version.present? && view.id != content_view_version.content_view_id
-        errors.add(:base, _("Invalid association of the content view id. Content View must match the content view version being saved"))
+        errors.add(:base, _("Invalid association of the content view id. Content View must match the content view version being saved."))
       end
 
       if view.composite?
-        errors.add(:base, _("Cannot add composite versions to a composite content view"))
+        errors.add(:base, _("Cannot add composite versions to a composite content view."))
       end
 
       if view.default?
-        errors.add(:base, _("Cannot add default content view to composite content view"))
+        errors.add(:base, _("Cannot add default content view to composite content view."))
       end
 
       if attached_content_view_ids.include?(view.id)
-        errors.add(:base, _("Duplicate Content View. Another component already includes this view"))
+        errors.add(:base, _("Another component already includes content view with ID " + view.id.to_s + "."))
```

Comment:
Including extra data (like the content view did here)  in a translated string requires a different approach so that translations continue to work correctly. It's called a format string:  https://www.dotnetperls.com/format-ruby Give it a try - you should be able to get the exact same output